web/sass-files/sass/partials/_post.scss

```diff
@@ -374,9 +374,9 @@ body.ios {
 	ul {
 		margin: 0;
 		padding: 0;
-		list-style: none;
```

Comment:
@crspeller Looks like you added this line when you were adding permalinks. Can you remember why?